target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1583,6 +1586,18 @@ static int rtl8367b_detect(struct rtl8366_smi *smi)
 
 	dev_info(smi->parent, "RTL%s chip found\n", chip_name);
 
+	if (of_get_property(smi->parent->of_node, "realtek,extif2", &size))
+		smi->cpu_port = RTL8367B_CPU_PORT_NUM + 2;
+	else if (of_get_property(smi->parent->of_node, "realtek,extif1", &size)) {
+		if (rvb) /* for the RTL8367R-VB chip, extif1 corresponds to cpu_port 5 */ 
+			smi->cpu_port = RTL8367B_CPU_PORT_NUM;
+		else
+			smi->cpu_port = RTL8367B_CPU_PORT_NUM + 1;
+	} else
+		smi->cpu_port = RTL8367B_CPU_PORT_NUM;
```

Comment:
```
		if (of_get_property(smi->parent->of_node, "realtek,extif1", &size) && !rvb)
			smi->cpu_port += 1;
		else if (of_get_property(smi->parent->of_node, "realtek,extif2", &size))
			smi->cpu_port += 2;
```

Reply:
But calling rtl8367_detect twice (or more) would change the value.